pages/docs/reference/classes.md

```diff
@@ -114,6 +114,22 @@ class Person(val firstName: String, val lastName: String, var age: Int) { /*...*
 
 </div>
 
+The Kotlin code style recommends using a [trailing comma](coding-conventions.html#trailing-commas) when you declare class properties.
+It simplifies reordering, deleting, and adding new properties. Using trailing commas are optional: without them your code will work as well.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
```

Comment:
Maybe it's better to rewrite the class from the example above to show only the syntax difference?
Now there are structure differences, so it's harder to find the difference we want to show.